types/secure-random-password/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for secure-random-password 0.1
+// Project: https://github.com/rackerlabs/secure-random-password
+// Definitions by: Bjørnar Snoksrud <https://github.com/bjornars>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export const lower: string;
+export const upper: string;
+export const consonants: string;
+export const vowels: string;
+export const digits: string;
+export const symbols: string;
+export const fullSymbols: string;
+export const copyableSymbols: string;
+
+export interface RandomPasswordOptions {
+    length?: number;
+    characters?: string[];
```

Comment:
The docs indicate that this accepts both a string and an array of strings.
```suggestion
    characters?: string | string[];
```

Reply:
You are of course correct. I verified with the implementation that it works as expected.